std/random.d

```diff
@@ -2772,7 +2772,23 @@ if (isRandomAccessRange!Range && hasLength!Range && isUniformRNG!RandomGen)
 }
 
 /// ditto
-auto ref choice(Range)(auto ref Range range)
+auto ref choice(Range)(Range range)
```

Comment:
Constraints never existed here actually. I don't know why, but I think because it's calling a constrained overload and they will be applied anyway. For consistency, I removed constraints from a similar overload that I added

Reply:
OK. I had a slight concern that somehow the missing constraints were having an effect on overloading, but since you added the unit test on the constrained call, I think it's good.